packages/frontend/package.json

```
@@ -55,11 +56,11 @@
   "devDependencies": {
     "@tsconfig/node20": "^20.1.4",
     "@types/node": "^18.11.18",
-    "@types/reach__router": "1.3.6",
+    "@types/reach__router": "^1",
     "@types/react": "^18.0.15",
     "@types/react-dom": "^18.0.6",
     "@vitejs/plugin-react-refresh": "1.3.6",
-    "react-bootstrap-icons": "1.3.0",
+    "react-bootstrap-icons": "^1.11.5",
```

Is update to react-bootstrap-icons required to bump to React v19?
```suggestion
    "react-bootstrap-icons": "1.11.5",
```

If yes, use fixed version

---

Not necessarily. Just updated to keep it up-to-date. But older version also works.